cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -480,6 +480,50 @@ a|
 Terminate transaction now allows `YIELD`.
 The `WHERE` clause is not allowed on its own, as it is for `SHOW`, but needs the `YIELD` clause.
 
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+SHOW TRANSACTIONS [transaction-id[,...]]
+----
+[source, cypher, role="noheader"]
+----
+TERMINATE TRANSACTIONS transaction-id[,...]
+----
+a|
+The `transaction-id` now allows general expressions resolving to a string or a list of strings instead of just parameter.
+
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+SHOW TRANSACTIONS [transaction-id[,...]]
+YIELD field[, ...]
+  [ORDER BY field[, ...]]
+  [SKIP n]
+  [LIMIT n]
+  [WHERE expression]
+TERMINATE TRANSACTIONS transaction-id[,...]
+YIELD field[, ...]
+  [ORDER BY field[, ...]]
+  [SKIP n]
+  [LIMIT n]
+  [WHERE expression]
+RETURN field[, ...]
+  [ORDER BY field[, ...]]
+  [SKIP n]
+  [LIMIT n]
+----
+a|
+The `SHOW` and `TERMINATE TRANSACTIONS` commands can be combined in the same query.
+The order of the commands does not matter and there can be more than one of each command.
```

Comment:
Not sure if that is the correct word either. The result might differ depending on the order (and your privileges) but they can be used in any order. Not sure what word do fit 🤔

Reply:
Hmmm right. So maybe "matter" could work or maybe a rephrasing like "The command does not require a specific order and there can be more than one of each command"?

Reply:
Maybe something like?
```suggestion
The query does not require a specific order and there can be zero or more of each command type.
```

Reply:
The way it is written now, I understand that the query can have zero commands? Is that possible? If not, maybe it would make more sense to be like this? 

```suggestion
The query does not require a specific order and there can be more than one of each command.
```

Reply:
This sentence is not easy 😬

Reply:
I agree! 🥲 What about this? 

```suggestion
The query does not require a specific order or a minimum number of commands.
```

Or then completely remove the mention to the amount of commands, or is this important information that must be explicit?

```suggestion
The query does not require a specific order.
```

Reply:
I don't think it needs to be explicit but it felt more correct to have something about it. But your first sentence still looks like we allow 0 commands total which is wrong.
```suggestion
The query does not require a specific order and there can be zero or more of each command type, however at least one command is needed.
```

Reply:
I guess now it's much better! Thanks for the suggestion!